charged-ieee/template/main.typ

```diff
@@ -37,9 +38,48 @@ To demonstrate the potential of Typst, we conducted a series of experiments comp
 
 Overall, we believe that Typst represents a significant step forward in the field of scientific writing and typesetting, providing researchers with a valuable tool to streamline their workflow and focus on what really matters: their research. In the following sections, we will introduce Typst in more detail and provide evidence for its superiority over other typesetting systems in a variety of scenarios.
 
-= Methods
-#lorem(90)
+= Methods <sec:methods>
+#lorem(45)
 
-$ a + b = gamma $
+$ a + b = gamma $<formula>
+
+#lorem(80)
+
+#figure(
+  placement: none,
+  circle(width: 21%),
+  caption: [A circle representing the Sun.] // Centered under the image
+)<fig:sun>
+
+In @fig:sun you can see a common representation of the Sun, which is a star that is located at the center of the solar system.
+
+#lorem(80)
+
+#figure(
+  caption: [The Planets of the Solar System and Their Average Distance from the Sun],
+  placement: none,
+  table(
+    columns: (auto, auto),
+    align: (col, row) => (left, right).at(col),
+    inset: (x:12pt, y:4pt),
+    stroke: none,
+    fill: (x, y) => if y > 0 and calc.rem(y, 2) == 0  { rgb("#efefef") },
+    table.header(
+      [Planet], [Distance (million km)]
+    ),
+    table.hline(stroke:.5pt),
```

Comment:
should that really be part of the _document?_ wouldn't that be more appropriate for the template? there's also some missing spaces in there.

Reply:
Table style does not appear to be specified by IEEE. This is just _one_ example of how to make a somewhat pretty-looking table or at least it demonstrates some of the knobs a user may want to turn to fit their particular table data.